app/templates/views/dashboard/_totals.html

```diff
@@ -1,43 +1,25 @@
 <div class="ajax-block-container">
   <div class="grid-row grid-gap">
     <div id="total-sms" class="grid-col-12">
-      <span class="big-number-with-status">
-        <a class="usa-link display-flex" href="{{ url_for('.view_notifications', service_id=service_id, message_type='sms', status='sending,delivered,failed') }}">
+      <span class="big-number-with-status display-block margin-bottom-2">
+        <p>
```

Comment:
this p tag could be deleted. It's creating this large gap between the number and "Messages sent"
![image](https://github.com/GSA/notifications-admin/assets/53159604/bb57fc15-a4d3-4e49-8f6b-d1c42ffe92d5)